### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tiagofreire-pt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ in hot spots where precision can be leveraged safely.
 ## Dependencies ##
 
 * **[Go](https://golang.org/)** (1.11+) must be installed from a system package or manually.
+* Please make sure that OpenJDK (11 preferred), socat and git are installed on the OctoPrint device. For raspbian, this can be done with: ```sudo apt install openjdk-11-jdk-headless socat git-core```
 
 ## Marlin Configuration ##
 * Update with [current compatible branch](hhttps://github.com/MarlinFirmware/Marlin).
```
Comment (bwiedmann) [inline README.md]:
```suggestion
* Update with [current compatible branch](https://github.com/MarlinFirmware/Marlin).
```

**Review by bwiedmann — COMMENTED** [review]
Can you explain why these dependencies are needed?

**Comment by colinrgodsey** [comment]
@bwiedmann hmm, Java is actually no longer needed for the new revision. Go is needed for compiling the new version, as it's distributed by source and built locally by the octoprint plugin.

File: README.md [hunk 2]
```diff
@@ -25,9 +25,10 @@ in hot spots where precision can be leveraged safely.
 ## Dependencies ##
 
 * **[Go](https://golang.org/)** (1.11+) must be installed from a system package or manually.
+* Please make sure that OpenJDK (11 preferred), socat and git are installed on the OctoPrint device. For raspbian, this can be done with: ```sudo apt install openjdk-11-jdk-headless socat git-core```
```
Comment (colinrgodsey) [inline README.md:28]: these are actually not needed for the new version, we just need a golang install available

File: README.md [hunk 3]
```diff
@@ -25,9 +25,10 @@ in hot spots where precision can be leveraged safely.
...
 
 ## Marlin Configuration ##
-* Update with [current compatible branch](hhttps://github.com/MarlinFirmware/Marlin).
+* Update with [current compatible branch](https://github.com/MarlinFirmware/Marlin).
```
Comment (colinrgodsey) [inline README.md:31]: woops

**Pull request closed without merging**